api/v1/users/PKPUserHandler.inc.php

```diff
@@ -263,6 +265,7 @@ private function _buildListRequestParams($slimRequest) {
 	private function _buildReviewerListRequestParams($slimRequest) {
 
 		$returnParams = $this->_buildListRequestParams($slimRequest);
+		$contextId = $returnParams['contextId'];
```

Comment:
I think this warning could instead be resolved by removing https://github.com/pkp/pkp-lib/pull/5306/files#diff-22f1b14d85eac6760b01a79df3eef5ffR294 (and this new line)... I don't see any way `$returnParams['contextId']` can be overridden (per https://github.com/pkp/pkp-lib/pull/5306/files#diff-22f1b14d85eac6760b01a79df3eef5ffR293) and `$contextId` isn't otherwise used.